src/Illuminate/Cookie/Middleware/EncryptCookies.php

```diff
@@ -35,10 +35,10 @@ class EncryptCookies
     /**
      * Create a new CookieGuard instance.
      *
-     * @param  \Illuminate\Contracts\Encryption\Encrypter  $encrypter
+     * @param  \Illuminate\Encryption\EncryptionManager  $encrypter
      * @return void
      */
-    public function __construct(EncrypterContract $encrypter)
+    public function __construct(EncryptionManager $encrypter)
```

Comment:
Surely this should me an Encryption Factory contract like for cache?

Reply:
Added a new `Factory` contract for the `EncryptionManager` and updated references to `EncryptionManager` to point to this `Factory`